src/fireflyClient.ts

```diff
@@ -95,6 +95,8 @@ export class FireflyClient {
 
   private maxBlockGasLimit = 0;
 
+  private marginBankPrecision = 6;
```

Comment:
change name to `MarginTokenPrecision` aur upar comment daal do yahan: `// the number of decimals supported by USDC contract`